### PR TITLE
Add Status management image diffs

### PR DESCRIPTION
**Pull request by shanemc-goog**
### Description

Add image diffs for status management page.

## Release notes:

NA - testing

### Checklist

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)

Fixes #<issue_number>; Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

File: browser-test/src/civiform_admin_program_statuses.test.ts [hunk 1]
```diff
@@ -38,6 +41,15 @@ describe('modify program statuses', () => {
       await ctx.adminPrograms.gotoDraftProgramManageStatusesPage(programName)
     })
 
+    it('renders create new status model', async () => {
```
Comment (nbeloglazov) [inline browser-test/src/civiform_admin_program_statuses.test.ts:44]: nit: modAl

**Review by nbeloglazov — APPROVED** [review]

File: browser-test/src/civiform_admin_program_statuses.test.ts (hunk 1 above)
Comment (shanemc-goog) [reply browser-test/src/civiform_admin_program_statuses.test.ts]: done!

**Comment by shanemc-goog** [comment]
PTAL, I accidentally muddied the PR with similar changes for another file and can't undo it.

**Comment by nbeloglazov** [comment]
I see that we are are capturing applicant ID in screenshots which is not stable. In cases like these we mark such DOM elements with class like [`.cf-applicants-id`](https://github.com/civiform/civiform/blob/db5a39a18392beb1b8d1f72dd3fa5dd0bc324f98/server/app/views/style/ReferenceClasses.java#L134) which later [replaced with hardcoded value](https://github.com/civiform/civiform/blob/main/browser-test/src/support/index.ts#L458) when screenshot is taken. I think you'll have to do the same thing for applicant id in your screensshots.

**Review by nbeloglazov — APPROVED** [review]

**Comment by shanemc-goog** [comment]
> I see that we are are capturing applicant ID in screenshots which is not stable. In cases like these we mark such DOM elements with class like [`.cf-applicants-id`](https://github.com/civiform/civiform/blob/db5a39a18392beb1b8d1f72dd3fa5dd0bc324f98/server/app/views/style/ReferenceClasses.java#L134) which later [replaced with hardcoded value](https://github.com/civiform/civiform/blob/main/browser-test/src/support/index.ts#L458) when screenshot is taken. I think you'll have to do the same thing for applicant id in your screensshots.

I think you saw my update already, but I had to make the normalizing code also process sub frames, which is why the screenshots weren't processing the application view subframe.

**Pull request merged**